docs/runner/intro/configure.md

```diff
@@ -617,18 +617,24 @@ marathon {
 
 ### Output configuration
 
-#### Max file path
+#### Max file name length
 
-By default, the max file path for any output file is capped at 255 characters due to limitations of some OSs. This is the reason why some
-test runs have lots of "File path length cannot exceed" messages in the log. Since there is currently no API to programmatically
-establish this limit, it's the user's responsibility to set a larger value if the OS supports this and the user desires it.
+By default, the max file name length for any output file is capped at 255 characters due to limitations of most file systems. This is the 
+reason why some test runs have lots of "File name length cannot exceed" messages in the log. Since there is currently no API to 
+programmatically establish this limit, it's the user's responsibility to set a larger value if the OS supports this and the user desires it.
+
+:::info
```

Comment:
I guess it is related to a path not a file name

Reply:
Not really, the new error messages specifically separate filename length and file path length